status/views.py

```diff
@@ -57,7 +60,7 @@ def status(request, username=None):
             user = User.objects.get(username=username)
             submissions = submissions.filter(user=user)
         except:
-            raise Http404('User %s Not Found!' % username)
+            raise Http404
```

Comment:
I agree with you.. However, there is no message to tell users what's wrong when redirect to 404 page, I think it's better to inform users in case of we know what happened.
Any other suggestion?